cmd/operator/main.go

```diff
@@ -76,6 +79,9 @@ func init() {
 	flag.StringVar(&awsConfig, "backup-aws-config", "",
 		"The name of the kube configmap object that stores the AWS config file. The file name must be 'config'.")
 	flag.StringVar(&s3Bucket, "backup-s3-bucket", "", "The name of the AWS S3 bucket to store backups in.")
+	flag.StringVar(&absSecret, "backup-abs-secret", "",
```

Comment:
We will deprecate the the operator wide flag for backup storage options.
Each cluster's backup storage option should sit in its own cluster spec.